active-directory-dotnet-native-uwp-v2/active-directory-dotnet-native-uwp-v2.csproj

```diff
@@ -126,10 +126,10 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Identity.Client">
-      <Version>3.0.3-preview</Version>
+      <Version>3.0.4-preview</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
-      <Version>6.1.9</Version>
+      <Version>6.2.8</Version>
```

Comment:
Is this necessary? (that's ok, BTW)

Reply:
No, it is not required, but I like it when samples are up to date with the latest dependencies. Looks clean.